install.js

```diff
@@ -304,6 +305,14 @@ function requestBinary(requestOptions, filePath) {
   return deferred.promise
 }
 
+function getTargetPlatform(){
+  if (process.env.PHANTOMJS_PLATFORM){
```

Comment:
J/K: https://docs.npmjs.com/misc/config

I don't necessarily agree, the npm_config_VAR is meant to allow overriding of npm configuration properties, platform and arch aren't npm configuration properties.

furthermore, in this case we actually don't want to ignore process.platform for all purposes.. path handling should still be done as if on a linux machine; we just need to denote which non-portable binary you are pulling down which is phantomjs specific and not a function of node process.platform.